tests/foreman/cli/test_host.py

```diff
@@ -1637,6 +1638,35 @@ def test_positive_apply_errata(self):
 
     @tier3
     @run_only_on('sat')
+    def test_positive_apply_security_erratum(self):
+        """Apply security erratum to a host
+
+        :id: 4d1095c8-d354-42ac-af44-adf6dbb46deb
+
+        :Assert: erratum is recognized by the
+        `yum update --security` command on client
+
+        :CaseLevel: System
+        """
+        self.client.download_install_rpm(
+            FAKE_0_YUM_REPO,
+            FAKE_1_CUSTOM_PACKAGE
+        )
+        # downgrade walrus package
+        self.client.run('yum downgrade -y {0}'.format(
+            FAKE_1_CUSTOM_PACKAGE_NAME))
+        # apply sea erratum that is security advisory
+        Host.errata_apply({
+            u'errata-ids': FAKE_1_ERRATA_ID,
+            u'host-id': self.host['id'],
+        })
+        result = self.client.run(
```

Comment:
Is system always up to date after deployment and only one package (walrus in our case) will require update after downgrade?
If yes, it will be nice to check that we have one security errata to be applied before the process started